map_it_test.go

```diff
@@ -1071,6 +1063,145 @@ func TestMap_ExecuteOnEntries(t *testing.T) {
 	})
 }
 
+func TestMap_AddInterceptor(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		ctx := context.Background()
+		prefix := "My Prefix"
+		id, err := m.AddInterceptor(ctx, &MapGetInterceptor{Prefix: prefix})
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func() {
+			if _, err := m.RemoveInterceptor(ctx, id); err != nil {
+				log.Fatal(err)
+			}
+		}()
+		v, err := m.Get(ctx, "foo")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.Equal(t, prefix, v)
+		ok, err := m.RemoveInterceptor(ctx, id)
+		if err != nil {
+			log.Fatal(err)
+		}
+		assert.Equal(t, ok, true)
+		v, err = m.Get(ctx, "foo")
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.Equal(t, nil, v)
+	})
+}
+
+func TestMap_TryPut(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		ctx1 := m.NewLockContext(context.Background())
+		if err := m.Lock(ctx1, "foo"); err != nil {
```

Comment:
I think we forget to Unlock in this test ?

Reply:
Map names are random, so leaving out unlocking has no effect on this test. But added for completeness at: https://github.com/hazelcast/hazelcast-go-client/pull/648/commits/7c277e2402b453759283fb1c77ceca950daad082